server/utils/constants.js

```diff
@@ -131,6 +133,11 @@ const EVENTS = {
     SEND: 'websocket.send',
     SEND_ALL: 'websocket.send-all',
   },
+  HOUSE: {
+    CREATE: 'house.create',
+    UPDATE: 'house.update',
+    DELETE: 'house.delete',
```

Comment:
As it's a result, and to be consistent with other variables, I would change the tense here.

HOUSE. CREATED: 'house.created'
HOUSE. UPDATED: 'house.updated'
HOUSE. DELETED: 'house.deleted'